lib/lf/uscalfe/fe_tools.h

```diff
@@ -200,7 +200,7 @@ static const unsigned int kout_prj_vals = 2;
  * @snippet fe_tools.cc nodalProjection
  */
 template <typename SCALAR, typename MF, typename SELECTOR = base::PredicateTrue>
-auto NodalProjection(const UniformScalarFESpace<SCALAR> &fe_space, MF &&u,
+auto NodalProjection(const ScalarFESpace<SCALAR> &fe_space, MF &&u,
```

Comment:
All functions and utilities that can work with `ScalarFESpace` should probably be moved to another namespace.